charts/manifests/templates/utils/_common.tpl

```diff
@@ -49,3 +50,60 @@ limitations under the License.
     {{- fail "Template requires '.context' as arguments" }}
   {{- end }}
 {{- end -}}
+
+
+{{/*
+  Sprig Template - SelectorLabels Wrapper. Adds Bundlename to selectorLabels, if defined.
+*/}}
+{{- define "bedag-lib.utils.common.selectorLabels" -}}
+{{- $c := . -}}
+{{- include "lib.utils.common.selectorLabels" $c }}
+{{- if .bundlename }}
+app.kubernetes.io/bundle:  {{ .bundlename }}
+{{- end }}
+{{- end -}}
+
+
+{{/*
+  Sprig Template - DefaultLabels Wrapper. Adds Bundlename to DefaultLabels, if defined.
+*/}}
+{{- define "bedag-lib.utils.common.defaultLabels" -}}
+{{- include "bedag-lib.utils.common.selectorLabels" . }}
+{{- if and .Chart.AppVersion (not .versionunspecific) }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+{{- end -}}
+
+
+{{/*
+  Sprig Template - OverwriteLabels Wrapper. Adds Bundlename to OverwriteLabels, if defined.
+*/}}
+{{- define "bedag-lib.utils.common.overwriteLabels" -}}
+  {{- if and $.Values.overwriteLabels (kindIs "map" $.Values.overwriteLabels) }}
+    {{- include "lib.utils.strings.template" (dict "value" $.Values.overwriteLabels "context" $) | nindent 0 }}
+  {{- else }}
+    {{- include "bedag-lib.utils.common.defaultLabels" . | indent 0}}
+  {{- end }}
+{{- end -}}
+
+
+{{/*
+  Sprig Template - CommonLabels Wrapper. Adds Bundlename to CommonLabels, if defined.
+*/}}
+{{- define "bedag-lib.utils.common.commonLabels" -}}
+  {{- include "bedag-lib.utils.common.overwriteLabels" . | indent 0 }}
+  {{- if and $.Values.commonLabels (kindIs "map" $.Values.commonLabels) }}
+    {{- include "lib.utils.strings.template" (dict "value" $.Values.commonLabels "context" $) | nindent 0 }}
+  {{- end }}
+{{- end -}}
+
+
+{{/*
+  Sprig Template - Labels Wrapper. Adds Bundlename to Labels, if defined.
+*/}}
+{{- define "bedag-lib.utils.common.labels" -}}
+  {{- $_ := set (default . .context) "versionunspecific" (default false .versionUnspecific ) -}}
+  {{- toYaml (mergeOverwrite (fromYaml (include "bedag-lib.utils.common.commonLabels" (default . .context))) (default dict .labels)) | indent 0 }}
+  {{- $_ := unset (default . .context) "versionunspecific" }}
+{{- end -}}
+
```

Comment:
Why would you duplicate all that code from the base library, when you just want to add one label?